app/components/facilities/FacilityListMapViewComponent.ios.js

```diff
@@ -47,7 +47,7 @@ const FacilityListMapViewComponent = (props) => {
   return (
     <View style={{flexGrow: 1}}>
       <TagScrollBarComponent tags={Tag.getAll()} onToggleFilter={updateFacilityList} hasInternet={props.hasInternet} type={'tag'}/>
-      <View style={{bottom: 256, position: 'absolute', zIndex: 1, flexGrow: 0, width: '100%'}}>
+      <View style={{bottom: Dimensions.get('screen').height <= 852 ? 268 : 256 , position: 'absolute', zIndex: 1, flexGrow: 0, width: '100%'}}>
```

Comment:
@limkimsan What does screen height 852 refer to? better assign a constant variable name to let us understand for the future maintenance.